ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable.yaml

```diff
@@ -121,7 +122,7 @@ tests:
       TEST_IMPORTANCE: Critical
     test:
     - chain: openshift-e2e-test-hypershift-qe
-    workflow: rosa-aws-sts-hypershift
+    workflow: rosa-aws-sts-hypershift-sector
```

Comment:
Perhaps also need to update the test name to reflect this change ?

Reply:
sure

Reply:
Ah, we will replace existing workflow to enable sector by default. The job name will only need to reflect the env. There will no non-sector test jobs currently in prod env. @LiangquanLi930  WDYT?

Reply:
I think we should better change to `aws-rosa-sts-hypershift-sector-prod-guest-p1-f7`, BTW don't forget update cron by `generate-cron-entry.sh`

Reply:
thanks. done

Reply:
The test name is updated, but diff from the suggested one. Would you mind to ack if this is expected, or update accordingly if it's not ?